tests/starship/tests/go.mod

```diff
@@ -0,0 +1,173 @@
+module github.com/cosmos-sdk/tests/starship/tests
+
+go 1.20
+
+replace github.com/strangelove-ventures/lens => github.com/Anmol1696/lens v0.1.1-0.20230705212610-c00628a886a0
+
+require (
+	github.com/cosmology-tech/starship/clients/go/client v0.0.0-20230715183507-198fb2b3ee2e
+	github.com/cosmos/cosmos-sdk v0.47.3
+	github.com/strangelove-ventures/lens v0.0.0-00010101000000-000000000000
```

Comment:
Could we remove this dep?

Reply:
We just need something to sign and broadcast txns. Is there something else I can use? Or maybe directly a client I can use?